app/views/providers/means_reports/_capital_disregards.html.erb

```diff
@@ -0,0 +1,17 @@
+<h2 class="govuk-heading-m"><%= t(".heading") %></h2>
+<%= govuk_summary_list(
```

Comment:
Summary card dependent on which branch is merged first, cc: @agoldstone93

Reply:
I'm assuming this will go in first, as testing will take time. I'm happy to make the changes on my branch once this is done

Reply:
yeah, i was thinking the same. I was guessing that it should only involve a couple of file changes on your branch and  would quite like to pair to see how this pans out as have not been looking into the card changes.